### DISCUSSION TITLE
GPIO_META_DET testpoint

### DISCUSSION THREAD

**Discussion by progzone122** [moto-penangf/discussions#6] [META] [answered]
It looks like there is a testpoint on the board for force transition to meta mode called GPIO_META_DET.
I've updated the documentation, it might come in handy someday (although we can go to META via Android Utility)

**Comment by shomykohai** [comment, marked as answer]
@progzone122 from the logs I found this:

`FAST META GPIO: 178`

And looking at moto g24 schematic, this point is called Fast META, and I found this src on github: https://github.com/hyperion70/HSPA_MOLY.WR8.W1449.MD.WG.MP.V16/blob/master/service/sys_svc/bootloader/src/bl_Main.c#L1901-L1912

So yeah, it sends us to META

**Reply by progzone122** [reply]
Don't forget that before this I was experimenting with meta, and past logs are not cleared!

**Reply by shomykohai** [reply]
I took this from my logs, I've sent the message before you sent the logs, but github is lagging https://www.githubstatus.com/
